docs/resources/artifactory_access_token.md

```diff
@@ -0,0 +1,173 @@
+# Artifactory Access Token Resource
+
+Provides an Artifactory Access Token resource. This can be used to create and manage Artifactory Access Tokens.
+
+~> **Note:** Access Tokens will be stored in the raw state as plain-text. [Read more about sensitive data in
+state](https://www.terraform.io/docs/state/sensitive-data.html).
+
+
+## Example Usages
+### Create a new Artifactory Access Token for an existing user
+```hcl
+resource "artifactory_access_token" "exising_user" {
+  username          = "existing-user"
+  end_date_relative = "5m"
+}
+```
+
+### Creates a new token for groups
+```hcl
```

Comment:
Please clarify is this creating a new user in a group or not?

Reply:
@peters95 @chb0github, Yes, this is creating a transient user in the group.
I've updated the documentation to explain.